test/scalacheck/src/test/scala/Ctrie.scala

```diff
@@ -42,19 +41,23 @@ object CtrieTest extends Properties("concurrent.TrieMap") {
       }
     }
 
+    val threads = for (idx <- 0 until totalThreads) yield new ParThread(idx)
```

Comment:
Interesting. Is that a change required by Dotty or just a coding style?

Reply:
Structural type support is different in Dotty: http://dotty.epfl.ch/docs/reference/changed/structural-types.html In general it's better to not use them if possible in Scala 2 since the generated code uses Java reflection.

Reply:
Oh indeed I didn’t notice the use of structural typing